### PR TITLE
feat(PE-7067): release name

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

File: src/state/actions/dispatchANTInteraction.ts [hunk 1]
```diff
@@ -117,6 +117,25 @@ export default async function dispatchANTInteraction({
...
+          name: payload.name,
+          ioProcessId: payload.ioProcessId,
+        });
+        dispatchSigningMessage('Verifying Release, please wait...');
+        const released = await arioContract
+          .getArNSRecord({ name })
+          .catch((e: Error) => e);
+
+        if (released instanceof Error) {
+          throw new Error('Failed to realease ArNS Name: ' + released.message);
```
Comment (kunstmusik) [inline src/state/actions/dispatchANTInteraction.ts:135]: Typo: "release"

File: .eslintrc [hunk 2]
```diff
@@ -35,8 +35,10 @@
     "no-unused-vars": "off",
     "@typescript-eslint/no-unused-vars": ["error", {}],
     "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
```
[the remark this answers is not in the document]
Comment (atticusofsparta) [reply .eslintrc:38]: This rule so far has been more annoying than it is helpfull

File: src/state/actions/dispatchANTInteraction.ts (hunk 1 above)
Comment (atticusofsparta) [reply src/state/actions/dispatchANTInteraction.ts]: [0508abb](https://github.com/ar-io/arns-react/pull/594/commits/0508abbfbeb54b3d2b596cd0d5b1130115fe4a0c)

File: .eslintrc (hunk 2 above)
[the remark this answers is not in the document]
Comment (kunstmusik) [reply .eslintrc:38]: More concerned about exhaustive-deps. I'll approve this PR but this should be revisited.

**Review by kunstmusik — APPROVED** [review]

**Pull request merged**